lib/active_record/connection_adapters/redshift/schema_definitions.rb

```diff
@@ -2,21 +2,6 @@ module ActiveRecord
   module ConnectionAdapters
```

Comment:
We moved the primary_key method from the TableDefinition class into the ColumnMethods module. Since they are all defined here is quite hard to follow.